developer-experience/recipes/remote-debugging/kubernetes-dashboard/kubernetes-dashboard.md

```diff
@@ -0,0 +1,42 @@
+# Remote Debugging using the Kubernetes Dashboard
+
+This document covers the Kubernetes Dashboard which is a useful tool for remote debugging for Kubernetes Clusters. It allows the management of applications running in the cluster, debug them and manage the cluster all through this dashboard.
```

Comment:
nit: remove for? so that it reads: This document covers the Kubernetes Dashboard which is a useful tool for remote debugging Kubernetes Clusters.

Reply:
you already have it under References but I am wondering if at this top level we should make `Kubernetes Dasboard` a link to https://kubernetes.io/docs/tasks/access-application-cluster/web-ui-dashboard/

Reply:
Done, thank you